stories/core/SkipToContentLink.stories.tsx

```diff
@@ -0,0 +1,191 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { Story, Meta } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import React from 'react';
+import {
+  SkipToContentLink,
+  SkipToContentLinkProps,
+  SkipToContentMarker,
+  Header,
+  HeaderLogo,
+  HeaderBreadcrumbs,
+  HeaderButton,
+  Body,
+  Kbd,
+  Anchor,
+} from '../../src/core';
+import { CreeveyStoryParams } from 'creevey';
+
+export default {
+  component: SkipToContentLink,
+  argTypes: {
+    className: { control: { disable: true } },
+    style: { control: { disable: true } },
+  },
+  title: 'Core/SkipToContentLink',
+  parameters: {
+    creevey: {
+      captureElement: null,
```

Comment:
It seems that all of the content is shown even when this parameter is removed

Reply:
Removed